python/giotto_json_reader.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+
+
+def cells_json(data):
+    '''
+    >>> data = {
+    ...     "cell_1": {
+    ...         "locations": [1632.02, -1305.7],
+    ...         "factors": {
+    ...             "pleiden_clus": [3],
+    ...             "kmeans": [8]
+    ...         },
+    ...         "mappings": {
+    ...             "tsne": [-11.0776, 6.0311],
+    ...             "umap": [-6.858, 15.7691]
+    ...         }
+    ...     },
+    ...     "cell_2": {
+    ...         "locations": [1589.47, -669.51],
+    ...         "factors": {
+    ...             "pleiden_clus": [3],
+    ...             "kmeans": [8]
+    ...         },
+    ...         "mappings": {
+    ...             "tsne": [-10.7971, 4.0234],
+    ...             "umap": [-8.0222, 16.2033]
+    ...         }
+    ...     }
+    ... }
+    >>> cells = cells_json(data)
+    >>> cells.keys()
+    dict_keys(['cell_1', 'cell_2'])
+    >>> cells['cell_1'].keys()
+    dict_keys(['mappings', 'genes', 'xy', 'factors', 'poly'])
+    >>> cells['cell_1']['mappings'].keys()
+    dict_keys(['t-SNE', 'UMAP'])
+    >>> cells['cell_2']['factors'].keys()
+    dict_keys(['pleiden_clus', 'kmeans'])
+
+    '''
+    cell_dict = {}
+    for cell in data.keys():
+
+        mappings_dict = {
+            't-SNE': data[cell]['mappings']['tsne'],
+            'UMAP': data[cell]['mappings']['umap']
+            }
+
+        genes_dict = {}
+
+        factors_dict = {
+            'pleiden_clus': "Cluster " +
+            str(data[cell]['factors']['pleiden_clus'][0]),
+            'kmeans': "Cluster " + str(data[cell]['factors']['kmeans'][0])
+            }
```

Comment:
Prefer `format` to concatenation:
```python
'pleiden_clus': 'Cluster {}'.format(data[cell]['factors']['pleiden_clus'][0]),
'kmeans': 'Cluster {}'.format(data[cell]['factors']['kmeans'][0])
```
(Style: stick with single quotes, usually.)